apps/api/src/app/invites/invites.controller.ts

```diff
@@ -30,17 +33,36 @@ export class InvitesController {
     @Post()
     @UseGuards(AuthGuard)
```

Comment:
We can remove `@UseGuards(AuthGuard)` because it can be used with an admin API Key.